transform/mattermost-analytics/models/marts/sales/hightouch/fct_marketing_campaign_member.sql

```diff
@@ -0,0 +1,38 @@
+with marketing as (
+    select marketing_id
+        , email as email
+        , subscribed_content
+        , server_id
+        , created_at
+        , updated_at
+    from
+        {{ ref('stg_cws__marketing')}}
+    qualify row_number() over (
+            partition by email
+            order by
+                created_at desc
+        ) = 1
+),
+marketing_campaign_member as (
+    select distinct m.marketing_id as marketing_id
+        , m.email as email
+        , m.server_id as server_id
+        , m.subscribed_content as subscribed_content
+        , l.lead_id as existing_lead_id
```

Comment:
The JIRA ticket also mentions a user id. Where should we get this data from?

Reply:
JIRA ticket mentions user id, but user id is not captured in cws data, this data is not available in the source and hence ignored.